benchmark/paddle/A2C/train.py

```diff
@@ -0,0 +1,196 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gym
+import numpy as np
+import os
+import time
+import parl
+from atari_model import AtariModel
+from atari_agent import AtariAgent
+from collections import defaultdict
+
+from parl.env.atari_wrappers import wrap_deepmind
+from parl.utils import logger, get_gpu_count, summary
+from parl.utils.scheduler import PiecewiseScheduler
+from parl.utils.time_stat import TimeStat
+from parl.utils.window_stat import WindowStat
+from parl.utils import machine_info
+
+from actor import Actor
+from parl.algorithms import A2C
+
+
+class Learner(object):
+    def __init__(self, config):
+        self.config = config
+
+        #=========== Create Agent ==========
+        env = gym.make(config['env_name'])
+        env = wrap_deepmind(env, dim=config['env_dim'], obs_format='NCHW')
+        obs_shape = env.observation_space.shape
+        act_dim = env.action_space.n
+        self.config['obs_shape'] = obs_shape
+        self.config['act_dim'] = act_dim
+
+        model = AtariModel(act_dim)
+        algorithm = A2C(model, vf_loss_coeff=config['vf_loss_coeff'])
+        self.agent = AtariAgent(algorithm, config)
+
+        if machine_info.is_gpu_available():
```

Comment:
Can we remove line 51~53? @zenghsh3

Reply:
It's unnecessary in the paddle dynamic version.